tests/utils/set_params.cpp

```diff
@@ -142,6 +142,9 @@ void setInvertParam(QudaInvertParam &inv_param)
   // Use 3D or 4D laplace
   inv_param.laplace3D = laplace3D;
 
+  if (Nsrc < Nsrc_tile || Nsrc % Nsrc_tile != 0)
```

Comment:
I think we should relax this sooner as opposed to later and put in the logic to let the last "tile" of sources be incomplete in the scope of this PR

Reply:
This is outside the scope of this PR.  To have different tiles sizes on each subset will very quickly break the autotuner, hence why we require this condition.  Fixing this is dependent on allowing sub-communicators to perform autotuning locally.